### PR TITLE
disable "inconsistent inner class entries" warning in quiltflower

### PR DESCRIPTION
**Pull request by eristhea**
this option to show this warning being enabled creates ~1700 lines of nonsense in the logs, it's best to just suppress it. This will make logs, especially actions logs, much more readable.

### CODE REVIEW

**Review by OroArmor — APPROVED** [review]

File: buildSrc/src/main/java/quilt/internal/decompile/quiltflower/QuiltflowerDecompiler.java
Comment (IotaBread) [inline buildSrc/src/main/java/quilt/internal/decompile/quiltflower/QuiltflowerDecompiler.java:43]: We could suppress these by filtering them out in `LoggerImpl#writeMessage` below

**Review by IotaBread — APPROVED** [review]

File: buildSrc/src/main/java/quilt/internal/decompile/quiltflower/QuiltflowerDecompiler.java
Comment (eristhea) [reply buildSrc/src/main/java/quilt/internal/decompile/quiltflower/QuiltflowerDecompiler.java:43]: true, but I think it would be better to not bandaid it and add an option to quiltflower
we're using bleeding-edge quiltflower anyway!

**Comment by eristhea** [comment]
update: I added a quiltflower option to disable the other warning (`inconsistent generic signature`), in https://github.com/QuiltMC/quiltflower/pull/233. We may want to wait on merging this until that's merged and we can use it.

**Comment by eristhea** [comment]
given that the other exception is fixable elsewhere, I think it's fine to merge and just make a follow-up to apply the fix in either draftsman or quiltflower

**Pull request merged**
